app/src/main/java/com/android/academy/fundamentals/solutions/MainActivity_Solution.kt

```diff
@@ -0,0 +1,43 @@
+package com.android.academy.fundamentals.solutions
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.android.academy.fundamentals.R
+import com.android.academy.fundamentals.SecondActivity
+import com.android.academy.fundamentals.solutions.SecondActivity_Solution.Companion.TRANSMITTED_BOOLEAN
+import com.android.academy.fundamentals.solutions.SecondActivity_Solution.Companion.TRANSMITTED_INT
+import com.android.academy.fundamentals.solutions.SecondActivity_Solution.Companion.TRANSMITTED_STRING
+
+class MainActivity_Solution : AppCompatActivity() {
```

Comment:
Have to rename because it can happen that students will import the wrong activity.